Tests/FuzzilliTests/VariableMapTest.swift

```diff
@@ -59,30 +60,33 @@ class VariableMapTests: XCTestCase {
         XCTAssertEqual(m1, m2)
         
         // Add another 128 elements and compare with a new map built up in the opposite order
-        for i in 128..<256 {
+        for i in 127..<255 {
             let val = Bool.random()
             m2[v(i)] = val
         }
         
         var m3 = VariableMap<Bool>()
         XCTAssertNotEqual(m1, m3)
-        
-        for i in (0..<256).reversed() {
+
+        for i in (0..<255).reversed() {
             m3[v(i)] = m2[v(i)] ?? false
         }
+
```

Comment:
Same, you can just revert all the previously existing test cases to how they were before. After all, the class should still behave the same way it did before :)

Reply:
Done, changes to the existing testcases have now been reverted to the original implementation.